boards/xtensa/m5stack_stamps3/Kconfig.board

```
@@ -0,0 +1,12 @@
+# M5Stack StampS3 board configuration
+
+# Copyright (c) 2023 Martin Kiepfer <mrmarteng@teleschirm.org>
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_M5STACK_STAMPS3
+	bool "M5Stack StampS3 IoT Development Board"
```

definitely a nit, but not sure about "IoT"

```suggestion
	bool "M5Stack StampS3 Development Board"
```

---

I agree. It's better to remove that. Saw that somewhere on the m5stack website.
Fixed in [38f0a59](https://github.com/zephyrproject-rtos/zephyr/commit/38f0a59c9701f755f1b41c0489c642aeb2e0bc7b)